src/components/toolbar/breadcrumb/Breadcrumb.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <nav id="breadcrumbs">
+    <ol v-if="breadcrumbData.length > 0" class="breadcrumb">
+      <li class="breadcrumb-item">
+        <a href="/"><i class="fas fa-home" /></a>
+      </li>
+      <li class="breadcrumb-item" v-for="(breadcrumb, index) in breadcrumbData[0]" :key="index">
+        <a :href="breadcrumb.url" v-if="breadcrumb.url">{{ breadcrumb.text }}</a>
+        <span v-if="!breadcrumb.url">{{ breadcrumb.text }}</span>
```

Comment:
Could this v-if be a v-else instead? That might save the negation of the property. Not sure if that makes it any clearer though.